pkg/storage/quota_pool.go

```diff
@@ -0,0 +1,81 @@
+package storage
+
+import (
+	"math"
+	"sync"
+
+	"golang.org/x/net/context"
+)
+
+const (
+	leaderProposalQuota   = 1000
```

Comment:
Probably. We'd like something that avoided Raft log truncation. This achieves that for the workload I'm using which involves small requests. More directly estimating the affect on Raft log size would be nice, though I'm not sure how to achieve that.